src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: bold;
```

Comment:
Don't do it here. Just set font weight to element general class. Also, move the font family style to the body tag.